tools/RAiDER/models/weatherModel.py

```diff
@@ -566,11 +566,10 @@ def getXY_gdal(self, filename):
         '''
         Pull the grid info (x,y) from a gdal-readable file
         '''
-        from osgeo import gdal
-        ds = gdal.Open(filename, gdal.GA_ReadOnly)
-        xSize, ySize = ds.RasterXSize, ds.RasterYSize
-        trans = ds.GetGeoTransform()
-        del ds
+        ds = rasterio.open(filename)
+        xSize, ySize = ds.width, ds.height
+        trans = ds.transform.to_gdal()
+        ds.close()
```

Comment:
I think this should use context manager:

```
with rasterio.open(filename) as ds:
    xSize, ySize = ds.width, ds.height
    trans = ds.transform.to_gdal()
```

Reply:
Done.